assets/packages/adminbasics/src/components/panelsanimation.js

```diff
@@ -10,7 +10,13 @@ export default function panelsAnimation() {
         /**
          * Card shown one by one
          */
-        document.querySelectorAll(".card").forEach(function (e, i) {
+        /** 
+         * @todo Added .welcome to restrict this to the welcome page (as it was intended), but the animation doesn't work
```

Comment:
Maybe just remove it ?

No animation can be really better for usage . It's not a game here.

Reply:
I would remove it on another ticket, focusing specially on that.
This is just for fixing the CPDB problem